### ISSUE TITLE
Some questions about paper

### ISSUE THREAD

**Issue by tomjerrygithub** [JirongZhang/DeepHomography#4]
Hi,Mr.Zhang ，i have some questions about the paper，can you explain it ？ thanks~
1、image input size is  315 × 560 ？
2、How to get Ga with Ma  and  Fa ?
3、And then ， np.dstack  Ga and Gb   to  Resnet34 ？
4、Training data is  imageA imageB and Homography matrix  from imageA to imageB ?
5、Can the training set be generated like 《Deep image homography estimation》?

I want to try to implement your paper, but I feel that there will be some difficulties in the loss function part, can I get your help?

**Comment by JirongZhang** [comment]
1. yes
2. pointwise multiply
3. torch.cat , do not use numpy
4. Hab =  imageA to imageB 
5. no
All processes need to be implemented with torch or tf, so that the entire calculation process is differentiable. You need to read ’ Unsupervised deep homography: A fast and robust homography estimation model ‘(https://arxiv.org/abs/1709.03966) first。
